subhub/featuretests/stripe_webhooks/test_stripe_webhook_calls.py

```diff
@@ -0,0 +1,73 @@
+from subhub.api.webhooks import stripeWebhookPipeline
+import json
+import os
+
+
+def test_stripe_webhook_charge_captured():
+    runTest("charge.captured.json")
+
+
+def test_stripe_webhook_charge_expired():
+    runTest("charge.expired.json")
+
+
+def test_stripe_webhook_disputeClosed():
+    runTest("charge.dispute.closed.json")
+
+
+def test_stripe_webhook_disputeCreated():
+    runTest("charge.dispute.created.json")
+
+
+def test_stripe_webhook_disputeFundsReinstated():
+    runTest("charge.dispute.funds_reinstated.json")
+
+
+def test_stripe_webhook_disputeFundsWithdrawn():
+    runTest("charge.dispute.funds_withdrawn.json")
+
+
+def test_stripe_webhook_disputeUpdated():
+    runTest("charge.dispute.updated.json")
+
+
+def test_stripe_webhook_failed():
+    runTest("charge.failed.json")
+
+
+def test_stripe_webhook_pending():
+    runTest("charge.pending.json")
+
+
+def test_stripe_webhook_refundUpdated():
+    runTest("charge.refund.updated.json")
+
+
+def test_stripe_webhook_refunded():
+    runTest("charge.refunded.json")
+
+
+def test_stripe_webhook_succeeded():
+    runTest("charge.succeeded.json")
+
+
+def test_stripe_webhook_updated():
+    runTest("charge.updated.json")
+
+
+def test_stripe_webhook_badpayload():
+    try:
+        runTest("badpayload.json")
+    except ValueError as e:
+        assert "this.will.break is not supported" == str(e)
+
+
+def runTest(fileName):
+    __location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+    f = open(os.path.join(__location__, fileName));
+
+    with f:
```

Comment:
why not ```with open(blah``` ?

Reply:
I have no idea

Reply:
http://cmdlinetips.com/2016/01/opening-a-file-in-python-using-with-statement/